rosa_learning/deploying_application_workshop/learning-lab-overview.adoc

```diff
@@ -0,0 +1,301 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="learning-lab-overview"]
+= Workshop overview
+include::_attributes/attributes-openshift-dedicated.adoc[]
+:context: learning-lab-overview
+toc::[]
+
+//rosaworkshop.io content metadata
+//Brought into ROSA product docs 22-JAN-2024
+//Modified for HCP 15 October 2024
+
+== Introduction
+
+After successfully provisioning your cluster, follow this workshop to deploy an application on it to understand the concepts of deploying and operating container-based applications. 
+
+.Workshop objectives
+
+* Deploy a Node.js-based application by using Source-to-Image (S2I) and Kubernetes deployment objects
+* Set up a continuous delivery (CD) pipeline to automatically push source code changes
+* Experience self healing applications
+* Explore configuration management through ConfigMaps, secrets, and environment variables
+* Use persistent storage to share data across pod restarts
+* Explore networking in Kubernetes and applications
+* Familiarize yourself with ROSA and Kubernetes functionality
+* Automatically scale pods based on loads from the Horizontal Pod Autoscaler
+* Use AWS Controllers for Kubernetes (ACK) to deploy and use an S3 bucket
+
+.Prerequisites
+
+* A provisioned ROSA cluster
+* The xref:../../cli_reference/openshift_cli/getting-started-cli.adoc#cli-getting-started[OpenShift command line interface (CLI)]
+* A link:https://github.com/signup[GitHub account]
+
+== About the OSToy application
+
+OSToy is a Node.js application that deploys to a ROSA cluster to help explore the functionality of Kubernetes.
+
+This application has a user interface where you can:
+
+* Write messages to the log (stdout / stderr)
+* Intentionally crash the application to view self-healing
+* Toggle a liveness probe and monitor OpenShift behavior
+* Read ConfigMaps, secrets, and environment variables
+* Read and write files when connected to shared storage
+* Check network connectivity, intra-cluster DNS, and intra-communication with the included microservice
+* Increase the load to view automatic scaling of the pods by using the Horizontal Pod Autoscaler
```

Comment:
Looked at style guide, looks like we are not formatting it like Kubernetes does from what I can tell....https://redhat-documentation.github.io/supplementary-style-guide/#horizontal-pod-autoscaler